### PR TITLE
Prepare 1.0.0-rc.2 release

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/account/operations/address_generation.rs [hunk 1]
```diff
@@ -18,14 +18,16 @@ use crate::events::types::{AddressData, WalletEvent};
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct AddressGenerationOptions {
     pub internal: bool,
-    pub metadata: GenerateAddressMetadata,
+    pub metadata: GenerateAddressOptions,
```
Comment (thibault-martinez) [inline src/account/operations/address_generation.rs:21]: Should we rename this to `options` as well ?

Comment (Thoralf-M) [reply src/account/operations/address_generation.rs:21]: That's done in https://github.com/iotaledger/wallet.rs/pull/1516

File: src/account/builder.rs [hunk 2]
```diff
@@ -214,7 +214,9 @@ pub(crate) async fn get_first_public_address(
             account_index,
             0..1,
             false,
-            GenerateAddressMetadata { syncing: true },
+            Some(GenerateAddressOptions {
+                ledger_nano_prompt: true,
+            }),
```
Comment (thibault-martinez) [inline src/account/builder.rs]: Should it be None instead ?

Comment (Thoralf-M) [reply src/account/builder.rs]: Yes, at least currently the behavior is like `None`

**Review by Thoralf-M — DISMISSED** [review]

**Review by samuel-rufi — DISMISSED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**